presto-docs/src/main/sphinx/release/release-347.md

```diff
@@ -13,6 +13,7 @@
 * Rename ``rewrite-filtering-semi-join-to-inner-join`` session property to ``rewrite_filtering_semi_join_to_inner_join``. ({issue}`5954`)
 * Throw a user error when session property value cannot be decoded. ({issue}`5731`)
 * Fix query failure when expressions that produce values of type `row` are used in a `VALUES` clause. ({issue}`3398`)
+* A minimum Java version of 11.0.7 is now required for Presto to start. This is to mitigate JDK-8206955. ({issue}`5957`)
```

Comment:
Can you add this to a new section "Server Changes"? That's what we've done in the past for other Java version changes.

Reply:
done